examples/de-aws-eks/jwt-public-key.pem

```diff
@@ -0,0 +1,9 @@
+-----BEGIN PUBLIC KEY-----
```

Comment:
Should we, perhaps, generate it and `.gitignore`? I feel so bad about publishing keys. Especially only part of the pair.